_posts/2023-11-16-virginia-beach-city/virginia-beach-city.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "
+Virginia Beach City"
+description: |
+  As one of many lawsuits against at-large voting systems for districts in local elections, Virginia Beach City recently adopted a new City Council district map that eliminates at-large seats. We explore the new implemented plan.
+author:
+  - name: Brian Zhou
+date: 2023-12-5
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+## The Virginia Beach City Council
+
+Virginia Beach became an incorporated town in 1906 from Princess Anne County, and became an independent city in 1952. A legal battle with the City of Norfolk prompted a consolidation of the City of Virginia Beach with Princess Anne County again in 1962, forming the independent city of Virginia Beach.
+
+Virginia Beach is heavily influenced by the presence of the military. Apart from the Pentagon in Arlington, Virginia, Virginia Beach has the largest concentration of military personnel with [9 military installations](https://www.yesvirginiabeach.com/business-environment/military) from every branch of the US armed forces; this includes the largest naval base in the world. Thus, it has been classified by the [American Communities Project](https://www.americancommunities.org/) as a military post: "middle-income, diverse communities, around military bases."
+
+There are 11 members in the Virginia Beach City Council. Ten members of the city council are elected in one of ten districts as shown in Figure 1. Displayed in Figure 2, three districts are currently Majority Minority Districts (MMDs), where the majority voting-eligible people in the district belong to a cohesive minority group.
+
+::: {style="display: flex;"}
+::: {style="flex-direction: column; margin-right: 5px;"}
+![Figure 1: *Virginia Beach City Council Districts*](citycouncil.svg){width="328"}
```

Comment:
I might be missing something since I'm rendering the file on my end, but what do the colors mean here? Is it just a different color for each district? I'd recommend adding in some more detail by conveying the minority percentage or partisan breakdown using the colors -- that way, you can still show the geography of the districts while giving insight into the racial/partisan makeup of the city.

Update: I see that you have a graph later showing the racial demographics. I would still say the colors here are a bit misleading, and it would make more sense to just have a single color across all districts if you're just trying to show the district boundaries.